lambdas/get_marker_request/get_marker_request_lambda_function.py

```diff
@@ -0,0 +1,55 @@
+import json
+import os
+import logging
+import boto3
+from data_service import DataService
+from location_marker import LocationMarker
+
+# Configure logging
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+# Initialize outside the handler for connection reuse
+dynamodb_resource = boto3.resource('dynamodb')
+
+def lambda_handler(event, context):
+    """
+    AWS Lambda handler function to retrieve a location marker.
+
+    :param event: AWS Lambda event object.
+    :param context: AWS Lambda context object.
+    :return: HTTP response with status code and body.
+    """
+    table_name = os.environ.get('TABLE_NAME')
+    if not table_name:
+        logger.error("TABLE_NAME environment variable is not set.")
+        return {
```

Comment:
In your return's you'll need to include the bellow `headers` to make the CORS work since our front end is somewhere else.

```
        return {
            'statusCode': 400,
            'headers': {
                'Access-Control-Allow-Origin': '*',  # Allow all origins for testing
                'Access-Control-Allow-Methods': 'GET,OPTIONS',  # Allowed methods
                'Access-Control-Allow-Headers': 'Content-Type',  # Allowed headers
            },
            'body': json.dumps({'error': 'Invalid request body.'})
        }
```